nav2_rviz_plugins/src/navigation_dialog.cpp

```diff
@@ -114,7 +115,12 @@ NavigationDialog::startNavigation(double x, double y, double theta, std::string
   pose.pose.position.z = 0.0;
   pose.pose.orientation = orientationAroundZAxis(theta);
 
-  action_client_->wait_for_action_server();
+  // wait for the action server to be available for 5 seconds
+  auto test = action_client_->wait_for_action_server(std::chrono::seconds(5));
```

Comment:
could we rename the variable from `test` to `is_server_ready`.

Reply:
I named it is_action_server_ready